website/docs/events/workshop-2023-07-10.md

```diff
@@ -0,0 +1,50 @@
+# Workshop: vmchecker: Automated Assignment Deployment, Checking and Grading
+
+Among the most difficult and time consuming activities in teaching / training are assignments, or practical evaluated items.
+An assignment consists of a statement and set of goals.
+Typically, assignments have a grading scale that ensure a fair grading scheme and also provides students / trainees with a set of steps for solving it.
+The deployment, checking and grading of assignments is a candidate for automation.
+Given the large number of submissions, having an automation engine taking care of this improves the work of teachers and also provides immediate feedback and direction to students.
+
+At [OpenEdu](https://open-education-hub.github.io/), we've developed and enhanced [vmchecker](https://github.com/systems-cs-pub-ro/vmchecker-next/wiki/Teaching-Assistant-Handbook), an engine to automate deployment, checking and grading of assignments.
```

Comment:
Maybe consider moving the repo under the OpenEdu organisation? Not sure if there's anything preventing that, but logically, it's an OpenEdu product, so I guess it should be there.

Reply:
@teodutu , yes, the repo will be moved. We'll do it after the workshop. Cc: @Sergiu121 , @Jokeswar